### DISCUSSION TITLE
Statamic Age Gate

### DISCUSSION THREAD

**Discussion by niteowlcreative** [statamic/cms#4544] [Q&A] [answered]
Good morning! 

I'm using the latest version of statamic and can find 'entrance gate' for Statamic2. But that is not the most up to date.

Is there a age-gate for Statamic 3 that can be used?

https://statamic.com/addons/ipx/entrance-gate

**Comment by edalzell** [comment, marked as answer]
Doesn't look like it. You can always add a Feature Request to their repo to ask.
